Sources/FetchNodeDetails/SampleOutput.swift

```diff
@@ -0,0 +1,152 @@
+//
```

Comment:
Don't bundle the sample files in main package. Keep them in correct folders

Reply:
Done